README.md

```diff
@@ -1 +1,108 @@
-PHP Library to perform 3 way merge on associative arrays.
+# Relaxedws/merge [![Build Status](https://travis-ci.org/relaxedws/merge.svg?branch=master)](https://travis-ci.org/relaxedws/merge)
+
+A Library to perform recursive 3 way merge algorithm
```

Comment:
Also, we should always use the correct punctuation for 3-way merge algorithm, i.e. use "-" between "3" and "way".